packages/admin/src/routes/entities/components/Action/actions/PasswordUpdate.js

```diff
@@ -0,0 +1,27 @@
+import {PasswordUpdateApp} from 'tocco-login/src/main'
+import {connect} from 'react-redux'
+import React from 'react'
+import PropTypes from 'prop-types'
+import {notification} from 'tocco-app-extensions'
+
+const mapStateToProps = state => ({
+  username: state.session.username
+})
+
+const mapActionCreators = {
+  closeModal: notification.removeModal
+}
+
+const PasswordUpdate = ({username, closeModal}) => {
+  return <PasswordUpdateApp
```

Comment:
`{ return ... }` not necessary

less boilerplate: `... => <PasswordUpdateApp...`